db/migrate/20231219123901_merge_folio_accounts_to_folio_users_and_drop_accounts_table.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class MergeFolioAccountsToFolioUsersAndDropAccountsTable < ActiveRecord::Migration[7.0]
+  def up
+    ActiveRecord::Base.connection_pool.with_connection do
+      sites = Folio::Site.all.to_a
+
+      accounts = connection.select_all("SELECT * FROM folio_accounts")
+
+      accounts.each do |account|
+        roles_to_pass = JSON.parse(account["roles"])
+        superadmin = roles_to_pass.delete("superuser").present?
+
+        user = Folio::User.find_by(email: account["email"])
+
+        if user.blank?
+          puts("Creating user for account #{account["email"]}")
+          except_attributes = %w[id roles is_active]
+          new_attrs = account.except(*except_attributes)
+          new_attrs["encrypted_password"] = account["encrypted_password"]
+          new_attrs["superadmin"] = superadmin
+          new_attrs["confirmed_at"] = Time.current
+          new_attrs["id"] = account["id"] unless Folio::User.exists?(id: account["id"])
```

Comment:
Tohle rozbije id sekvenci, ne? Radeji bych nahradil tech par mist, kde bylo `account_id`.

Reply:
Nehlede na to, ze je to k nicemu, kdyz to nebude 100% platit pro vsechny.